### PR TITLE
Calculate

### PR DESCRIPTION
**Pull request by pwiatr3**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @pwiatr3

**Comment by ziobron** [comment]
⏱ 3 XP extra for @pwiatr3 for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -3,5 +3,24 @@
...
+    else if(command == "multiply"){
+        return(std::to_string(first * second));
+    }
+    else if(command == "divide"){
+        if(second != 0){
+            return(std::to_string(first / second));
+        }
+        else {
+            return("Division by 0");
+        }
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:21]: If you return variable here, you don't need to use `else`

File: homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -3,5 +3,24 @@
...
+    }
+    else{
+        return "Invalid data";
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:24]: If you return variable here, you don't need to use `else`

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**